src/helpers/helpers.jsx

```diff
@@ -29,3 +28,29 @@ export const productDetails = (productArray) => {
     return resultString;
 };
 
+export const formatPrice = (productArray) => {
+    return `€${productArray.price},-`;
+};
+
+export const formatSizes = (productArray) => {
+    const inchesToCentimeters = 2.54;
+    let resultString = "";
+    productArray.availableSizes.forEach((size) => {
+        const sizeInInches = size;
+        const sizeInCentimeters = size * inchesToCentimeters;
+        resultString += `${sizeInInches} inches (${sizeInCentimeters.toFixed(2)} cm) | `;
+    });
+    return resultString.slice(0, -2);
```

Comment:
```suggestion
    const sizesStrings = productArray.availableSizes.map(size => {
        const sizeInInches = size;
        const sizeInCentimeters = size * inchesToCentimeters;
        return `${sizeInInches} inches (${sizeInCentimeters.toFixed(2)} cm)`;
    });
    
    return sizesStrings.join(" | ");
```
Iets nettere manier, zodat je niet op het eind nog een stukje eraf hoeft te halen 😉